pandas/io/parsers/readers.py

```diff
@@ -1367,9 +1367,21 @@ def read_fwf(
 
 class TextFileReader(abc.Iterator):
     """
+    Passed dialect overrides any of the related parser options.
 
-    Passed dialect overrides any of the related parser options
+    Iterator class used to process to text files read via read_csv in
+    chunks.
 
+    An instance of this class is returned by `read_csv` when it is processed in
+    chunks, instead of returning a single `DataFrame`.
+
+    When iterating over `TextFileReader`, every item returned will be a DataFrame.
+
+    Examples
+    ---------
+    >>> with pandas.read_csv(..., iterator=True) as text_file_reader:
```

Comment:
You need to import pandas or use `pd` which is imported by default to make the CI happy: https://github.com/pandas-dev/pandas/runs/7043600368?check_suite_focus=true#step:8:132